opensquirrel/__init__.py

```diff
@@ -1,4 +1,73 @@
 from opensquirrel.circuit import Circuit
 from opensquirrel.circuit_builder import CircuitBuilder
+from opensquirrel.default_gate_modifiers import ControlGateModifier, InverseGateModifier, PowerGateModifier
+from opensquirrel.default_instructions import (
+    CNOT,
+    CR,
+    CZ,
+    SWAP,
+    X90,
+    Y90,
+    CRk,
+    H,
+    I,
+    Rx,
+    Ry,
+    Rz,
+    S,
+    Sdag,
+    T,
+    Tdag,
+    X,
+    Y,
+    Z,
+    barrier,
+    default_bloch_sphere_rotation_set,
+    default_bloch_sphere_rotation_without_params_set,
+    default_unitary_set,
+    init,
+    measure,
+    mX90,
+    mY90,
+    reset,
+    wait,
```

Comment:
Do we want to sort these lines?

Reply:
Regarding this, would it be a good idea to run isort on the whole library? There are more unsorted imports in multiple files; isort would sort and format them alphabetically